pennylane/templates/embeddings.py

```diff
@@ -148,8 +159,14 @@ def BasisEmbedding(features, wires):
 
     .. note::
 
-        BasisEmbedding uses PennyLane's :class:`~pennylane.ops.BasisState` and only works in conjunction with
-        devices that implement this function.
+        ``BasisEmbedding`` uses PennyLane's :class:`~pennylane.ops.BasisState` and only works in conjunction with
+        devices that implement this operation.
```

Comment:
Same comment here: BasisEmbedding should automatically fallback to PauliX gates, so should work on any device.